api/src/main/java/com/google/sps/User.java

```diff
@@ -12,8 +12,8 @@
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
-    private String email;
-    private String username;
+    String email;
```

Comment:
Just curious, why were these fields changed to package-private instead of getters?

Reply:
you're right i should've used getters, gonna change that now